src/core/bluetoothdevicemodel.cpp

```diff
@@ -24,7 +24,7 @@ BluetoothDeviceModel::BluetoothDeviceModel( QObject *parent )
     mLocalDevice( std::make_unique<QBluetoothLocalDevice>() )
 {
   connect( &mServiceDiscoveryAgent, &QBluetoothServiceDiscoveryAgent::serviceDiscovered, this, &BluetoothDeviceModel::serviceDiscovered );
-  connect( &mServiceDiscoveryAgent, qgis::overload<QBluetoothServiceDiscoveryAgent::Error>::of( &QBluetoothServiceDiscoveryAgent::error ), this, [ = ]()
+  connect( &mServiceDiscoveryAgent, qOverload<QBluetoothServiceDiscoveryAgent::Error>( &QBluetoothServiceDiscoveryAgent::error ), this, [ = ]()
```

Comment:
I remember that when I wrote this line the tests failed since `qOverload` was a banned keyword. That's why I went for `qgis::overload` but I don't have the background to it and it seems to be fine here...

Reply:
Glad it's not anymore, otherwise we'd fail to compile against latest QGIS master :)